apps/dashboard/app/javascript/packs/batch_connect.js

```diff
@@ -716,21 +716,14 @@ function optionForFromToken(str) {
   document.getElementById(elementId).dispatchEvent((new Event('change', { bubbles: true })));
 };
 
-// simple function to sanitize css query strings
-function sanitizeQuery(item) {
-  return item.replaceAll('.', '\\.');
-}
-
-
-function optionForEvent(target) {
-  let simpleName = shortId(target['id']);
-  return mountainCaseWords(simpleName);
-};
-
```

Comment:
These removes (and the one above) aren't really a part of this pull request - I just noticed in my linter that they're unused.